.github/workflows/push_pr.yml

```diff
@@ -30,6 +30,13 @@ jobs:
       - name: Validate code
         run: make ci/validate
 
+  golangci-lint:
```

Comment:
oh i didn't know we are adding this linter. Do we keep ci/validate?

Reply:
I don't think so. I just removed it